.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-* @primetheus @zkoppert @admiralawkbar @migarjo @steffen
+* @primetheus @zkoppert @admiralawkbar @migarjo @steffen @dependabot-bot
```

Comment:
```suggestion
# Users that can approve pull requests
# @dependabot-bot needed for .github/workflows/auto-merge-depedabot.yml
* @primetheus @zkoppert @admiralawkbar @migarjo @steffen @dependabot-bot
```

@admiralAwkbar What do you think about adding ☝️ comments?

Reply:
i wanted to go big...

Reply:
and you did 😄👍